### PR TITLE
issue 736 fix for RT-3.1

### PR DESCRIPTION
**Pull request by Hobbydos**
Fix for [issue 736](https://github.com/openconfig/featureprofiles/issues/736)

### CODE REVIEW

File: feature/experimental/policy/policy_vrf_selection/ate_tests/base_vrf_selection/base_vrf_selection_test.go [hunk 1]
```diff
@@ -153,11 +153,11 @@ func configNetworkInstance(name string, peer *attrs.Attributes) *telemetry.Netwo
...
 
-	ni.Type = telemetry.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_L2L3
-	static := ni.GetOrCreateProtocol(telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, "STATIC")
-	ipv4Nh := static.GetOrCreateStatic("0.0.0.0/0").GetOrCreateNextHop(peer.IPv4)
+	ni.Type = telemetry.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_L3VRF
+	static := ni.GetOrCreateProtocol(telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, *deviations.StaticProtocolName)
+	ipv4Nh := static.GetOrCreateStatic("0.0.0.0/0").GetOrCreateNextHop("0")
 	ipv4Nh.NextHop, _ = ipv4Nh.To_NetworkInstance_Protocol_Static_NextHop_NextHop_Union(peer.IPv4)
-	ipv6Nh := static.GetOrCreateStatic("::/0").GetOrCreateNextHop(peer.IPv6)
+	ipv6Nh := static.GetOrCreateStatic("::/0").GetOrCreateNextHop("1")
```
[the remark this answers is not in the document]
Comment (Hobbydos) [reply feature/experimental/policy/policy_vrf_selection/ate_tests/base_vrf_selection/base_vrf_selection_test.go:160]: @robshakir, It was simply an internal test - Should be both set to `0` - I'll add a commit to set it back to `0`.

**Comment by xw-g** [comment]
LGTM too. pending @Hobbydos to update the OTG test before merging. 

cc: @vcheny @vosipchu @sachendras I think we can go ahead and merge with 2 LGTMs from @robshakir and I, because: 
* It does not introduce deviation. On contrary, it fixes a place that actually should use the deviation (reference: https://github.com/openconfig/featureprofiles/search?q=deviations.StaticProtocolName).
* It implements a agreed upon OC schema: https://github.com/openconfig/public/pull/732

Feel free to ping if objection.

**Comment by Hobbydos** [comment]
> LGTM too. pending @Hobbydos to update the OTG test before merging.
> 
> cc: @vcheny @vosipchu @sachendras I think we can go ahead and merge with 2 LGTMs from @robshakir and I, because:
> 
> * It does not introduce deviation. On contrary, it fixes a place that actually should use the deviation (reference: https://github.com/openconfig/featureprofiles/search?q=deviations.StaticProtocolName).
> * It implements a agreed upon OC schema: [Clarify static route index description to use a number public#732](https://github.com/openconfig/public/pull/732)
> 
> Feel free to ping if objection.

@xw-g 
Sure, I can commit the code to OTG test case, the only problem is that this current commit was created before the OTG test was published. Do you want me to rebase and push everything in this PR or create an additional PR?

**Comment by xw-g** [comment]
> 

We prefer same ATE and OTG tests in the same PR. 

Also seems like the base of your patch is not in sync with current main, and that leads to many diffs outside your current PR scope. @Hobbydos can you help fix that? Thanks.

**Pull request closed without merging**